### DISCUSSION TITLE
Show speed limit on VC from GPS data (not TSR)

### DISCUSSION THREAD

**Discussion by Jguog** [Mr-MIBonk/M.I.B._More-Incredible-Bash#107] [Ideas]
Hi. I am just amazed by the work being done with M.I.B. Alot of which I couldn't get working with OBDELEVEN after many years of searching.

This may be a far stretch, but would it even be remotely possible to enable VC to display speed limit from the built in maps data  (not TSR) using M.I.B.  I attempted many methods to enable TSR on my Audi in Australia, which all failed. However i have read somewhere that it is possible to have speed limit stored in maps data show up on VC. 

Really want to hear your thoughts on this.

![image](https://user-images.githubusercontent.com/36902676/217809373-da5ddcfe-4e8d-41a2-a09f-fe6b13a8edd6.png)

**Comment by harman-f** [comment]
There is no know method to change coding on cluster via M.I.B. 
so sadly, this is a no.

**Comment by ANMOSIN** [comment]
It remains only to dream that perhaps this will be implemented
